pytest_plugins/issue_handlers.py

```diff
@@ -42,14 +66,39 @@ def pytest_collection_modifyitems(session, items, config):
 
     pytest.issue_data = generate_issue_collection(items, config)
 
-    # No item deselect, just adding skipif markers
+    # Add skipif markers, and modify collection if there were BZ options passed
+    bz_filters = config.getoption('BZ', None)
```

Comment:
I m sure by default it should  be returning None, else we can set the default to none from pytest_addoption itself.

Reply:
It could, but I'd like to keep the current pattern instead of providing a default for `--BZ`.  The option is only set if the user provides it.